codegen/runner/runner.go

```diff
@@ -114,6 +114,9 @@ func main() {
 	defaultDependencies := make(map[string][]string, 0)
 	config.MustGetStruct("defaultDependencies", &defaultDependencies)
 
+	defaultHeaders := make([]string, 0)
+	config.MustGetStruct("defaultHeaders", &defaultHeaders)
```

Comment:
Meaning this is mandatory?  Unspecified = empty semantics hold?
Also, just asking -- since this is the place - any validation needed? i.e. disallowed values, values with spaces etc... just like any header ...

Reply:
good point, it shouldn't be must get, we should first check if the key exists and if it does, only then read it since this is an optional config.